libc/test.yaml

```diff
@@ -67,7 +67,7 @@ test:
       execute: test-libc-unix-socket
       targets:
         # Excluded due to long execution time
-        exclude: [armv7m4-stm32l4x6-nucleo]
+        exclude: [armv7m4-stm32l4x6-nucleo, armv7a9-zynq7000-qemu]
```

Comment:
I see, the issue with the tests taking too long relates to the test cases:` unix_msg_data_and_fd` and `unix_msg_fork`. Could we split each of these into two separate test cases, for example one for `SOCK_STREAM` and one for `SOCK_DGRAM`? This way, we could run them on every target, given that the issue here is simply that the Nucleo and Zynq-QEMU targets are slow.